style-guide.adoc

```diff
@@ -1516,60 +1527,61 @@ The table lists UI elements and the verbs and adjectives to use for them.
+
+==== Android-specific user interface elements
 
-=== Android user interface elements
+Check out the link:https://material.io/[Google Material Design resources] for the full list. 
 
-Check out Google material.io resources. 
+- *Bottom navigation bars:* Allow movement between primary destinations (3 to 5) in an app (mobile or tablet only). They consist of a container with icons that have text labels. Icons in the bottom navigation bars can be active or inactive. (note difference with unavailable). States: active, inactive, focused, or pressed.
 
-*Bottom navigation bars:* Allow movement between primary destinations (3 to 5) in an app (mobile or tablet only). They consist of a container with icons that have text labels. Icons in the bottom navigation bars can be active or inactive. (note difference with unavailable). States: active, inactive, focused, or pressed.
+- *Badges:* Bottom navigation icons can include badges in their upper right corner. These badges can contain dynamic information, such as a number of pending requests.
 
-*Badges:* Bottom navigation icons can include badges in their upper right corner. These badges can contain dynamic information, such as a number of pending requests.
+- *Top navigation bars:* Can contain menu icon (Hamburger), up arrow, back arrow, title, action items, and the overflow menu (3 dots)
 
-*Top navigation bars:* Can contain menu icon (Hamburger), up arrow, back arrow, title, action items, and the overflow menu (3 dots)
+- *Floating action buttons (FABs):* Performs the primary, or most common, action on a screen. FABs are displayed in front of all screen content, typically as a circular shape with an icon in their center. FABs come in three types: regular, mini, and extended.
 
-*Floating action buttons (FABs):* Performs the primary, or most common, action on a screen. FABs are displayed in front of all screen content, typically as a circular shape with an icon in their center. FABs come in three types: regular, mini, and extended.
+- **Text buttons:** Typically used for less-pronounced actions, including those located in dialogs and in cards. In cards, text buttons help maintain an emphasis on card content.
 
-**Text buttons:** Text buttons are typically used for less-pronounced actions, including those located:
-In dialogs, In cards. In cards, text buttons help maintain an emphasis on card content.
+- **Outlined buttons:** Medium-emphasis buttons that display a stroke around a text label. They contain actions that are important, but aren’t the primary action.
 
-**Outlined buttons:** Medium-emphasis buttons that display a stroke around a text label. They contain actions that are important, but aren’t the primary action in an app.
+- **Contained button:** Display a container around a text label. Contained buttons are high-emphasis, distinguished by their use of elevation and fill. They contain actions that are primary to an app.
```

Comment:
Instead of actions that are primary to an app, would it also be correct to say:  

They contain the primary actions of an app or screen.

actions that are primary is just a bit weird